examples/mobile/UIComponents/components/etc/themes/colors.html

```diff
@@ -22,7 +22,7 @@ <h1>
 			</h1>
 		</div>
 		<div class="ui-content">
-			<ul class="ui-listview" style="color: var(--color-white);">
+			<ul class="ui-listview ui-content-area" style="color: var(--color-white);">
 				<li style="background-color: var(--primary-color)" class="ui-li-flex">
```

Comment:
ditto.

Reply:
It's sample for color value checking.
So I think it don't need to go towards css.

Reply:
ok. I will consider it next PR.